### PR TITLE
8.0 ddt based on packaging preparation lb

### PR DESCRIPTION
**Pull request by eLBati**
Depends on https://github.com/OCA/stock-logistics-workflow/pull/159

### CODE REVIEW

File: l10n_it_ddt/__openerp__.py [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+#    GNU Affero General Public License for more details.
+#
+#    You should have received a copy of the GNU General Public License
+#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+##############################################################################
+
+{
+    'name': 'DDT',
+    'version': '1.0',
```
Comment (tafaRU) [inline l10n_it_ddt/__openerp__.py]: Please update it according to https://github.com/OCA/maintainer-tools/blob/master/CONTRIBUTING.md#version-numbers

File: l10n_it_ddt/__openerp__.py [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+    'category': 'Localization/Italy',
+    'summary': 'Documento di Trasporto',
+    'author': 'Davide Corio, Odoo Community Association (OCA),'
+              'Agile Business Group, Francesco Apruzzese, Apulia Software',
+    'website': 'http://www.odoo-italia.org/',
+    'license': 'AGPL-3',
+    'depends': [
+        'sale',
```
Comment (tafaRU) [inline l10n_it_ddt/__openerp__.py]: It's redundant, it would be better to remove it.

File: l10n_it_ddt/__openerp__.py [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+        'sale_stock',
+        'stock',
```
Comment (tafaRU) [inline l10n_it_ddt/__openerp__.py]: Same as above.

File: l10n_it_ddt/__openerp__.py [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+        'stock_account',
+        'stock_picking_package_preparation',
+        'stock_picking_package_preparation_line',
```
Comment (tafaRU) [inline l10n_it_ddt/__openerp__.py]: Same as above.

File: l10n_it_ddt/__openerp__.py [hunk 5]
```diff
@@ -0,0 +1,56 @@
...
+        'wizard/ddt_create_invoice.xml',
+        'views/stock_picking_package_preparation.xml',
+        'views/partner.xml',
+        'views/account.xml',
+        'views/sale.xml',
+        'wizard/add_picking_to_ddt.xml',
+        'wizard/ddt_from_picking.xml',
+        'views/report_ddt.xml',
+        ],
+    'test': [],
```
Comment (tafaRU) [inline l10n_it_ddt/__openerp__.py]: Please remove empty key.

**Comment by eLBati** [comment]
thanks @tafaRU I made the changes

File: l10n_it_ddt/data/ddt_data.xml [hunk 6]
```diff
@@ -0,0 +1,70 @@
...
+        </record>
+        <record id="transportation_method_DES" model="stock.picking.transportation_method">
+            <field name="name">DESTINATARIO</field>
+        </record>
+        <record id="transportation_method_COR" model="stock.picking.transportation_method">
+            <field name="name">CORRIERE</field>
+        </record>
+
+     </data>
+</openerp>
```
Comment (tafaRU) [inline l10n_it_ddt/data/ddt_data.xml]: No newline at end of file.

File: l10n_it_ddt/tests/new_ddt.yml [hunk 7]
```diff
@@ -0,0 +1,122 @@
+-
```
Comment (tafaRU) [inline l10n_it_ddt/tests/new_ddt.yml]: I dont't think this test is execued, insn'it?

File: l10n_it_ddt/wizard/add_picking_to_ddt.xml [hunk 8]
```diff
@@ -0,0 +1,41 @@
...
+            <field name="model_id" ref="stock_picking_package_preparation.model_stock_picking_package_preparation" />
+            <field name="name">Add Pickings to DDT</field>
+            <field name="key2">client_action_multi</field>
+            <field name="value" eval="'ir.actions.act_window,' + str(ref('action_pickings_to_ddt'))" />
+            <field name="key">action</field>
+            <field name="model">stock.picking</field>
+        </record>
+
+    </data>
+</openerp>
```
Comment (tafaRU) [inline l10n_it_ddt/wizard/add_picking_to_ddt.xml]: No newline at end of file.

File: oca_dependencies.txt [hunk 9]
```diff
@@ -1,2 +1,4 @@
 partner-contact
 account-payment
+account-invoice-reporting
```
Comment (tafaRU) [inline oca_dependencies.txt]: What's the reason for adding this?

**Comment by eLBati** [comment]
Thanks @tafaRU I made the changes

**Comment by andrea4ever** [comment]
All good for me,  :+1:

**Comment by OpenCode** [comment]
:+1:

**Comment by tafaRU** [comment]
Thanks @eLBati!
Could you also please use simple headers following OCA templates
https://github.com/OCA/maintainer-tools/blob/master/template/module/__openerp__.py#L1-L3 ?

Thank you in advance.

**Comment by eLBati** [comment]
@tafaRU done

**Comment by tafaRU** [comment]
Thanks @eLBati, I'll merge it once travis is green.

**Comment by tafaRU** [comment]
Coveralls buil is red due to https://github.com/eLBati/l10n-italy/commit/1f5e854aae432567f8ffa2fe43569a621972533e I merge then.

**Pull request merged**